README.md

```diff
@@ -58,14 +58,22 @@ Options:
 
 #### Linux
 
-Option 1 (recommended):
-Install the flatpak from [Flathub][flathub]: `flatpak --user install com.adilhanney.saber`.
+**Installing as a flatpak is recommended**
+
+<p>
+<a href='https://flathub.org/apps/details/com.adilhanney.saber'><img width='150' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a>
+</p>
+<p>
+  
+<a href="https://snapcraft.io/saber">
+  <img width='150' alt="Get it from the Snap Store" src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" />
```

Comment:
I'm not a fan of the badges as they mostly just link to the same install command we could directly link here. We can keep the badges but just above the existing written instructions.

Also please set the snap as Option 4 as per my previous comment: https://github.com/adil192/saber/pull/329#discussion_r1103290539

Reply:
> I'm unsure on the specifics of how snap sandboxing works. Please can you make sure that notes are written to the user's `~/Documents/Saber` folder like the flatpak does:
> 
> https://github.com/adil192/saber/blob/21171cac3d2bcc8d001ccd7d03c0f64a7c367d1f/flatpak/com.adilhanney.saber.json#L15

When the `home` plug is used, snap has the access of the whole home folder.